### PR TITLE
Revert Orchestrator changes

### PR DESCRIPTION
**Pull request by alivenichoppa**
### Change description ###
Revert orchestrator demo changes 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mdayican — COMMENTED** [review]
can you remove all env vars and secret settings, you can check from history Andrey added for his test.

**Comment by apereverzin** [comment]
@alivenichoppa 
From bulk-scan-orchestrator/demo.yaml - remove `keyVault`

**Review by mdayican — DISMISSED** [review]

**Review by apereverzin — DISMISSED** [review]

File: k8s/namespaces/bsp/bulk-scan-orchestrator/demo.yaml [hunk 1]
```diff
@@ -2,16 +2,11 @@ apiVersion: helm.fluxcd.io/v1
...
   name: bulk-scan-orchestrator
-  annotations:
-    fluxcd.io/automated: "true"
-    fluxcd.io/tag.java: glob:pr-1751-*
-    hmcts.github.com/prod-automated: disabled
 spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:pr-1751-8135623-20220301151447 #{"$imagepolicy": "flux-system:demo-bulk-scan-orchestrator"}
+      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:prod-79758fc-20220301130525 #{"$imagepolicy": "flux-system:demo-bulk-scan-orchestrator"}
```
Comment (mdayican) [inline k8s/namespaces/bsp/bulk-scan-orchestrator/demo.yaml:8]: I think you can remove `image`  as well

Comment (alivenichoppa) [reply k8s/namespaces/bsp/bulk-scan-orchestrator/demo.yaml]: Removed

**Review by mdayican — APPROVED** [review]

**Pull request merged**